medusa/helpers.py

```diff
@@ -1694,3 +1694,42 @@ def get_broken_providers():
 
     logger.info('Broken providers found: {0}'.format(response))
     return ','.join(response)
+
+
+def is_already_processed_media(full_filename):
```

Comment:
i kept this here as it can be used for something else other than providers/clients. 
Others i moved